README.md

```diff
@@ -0,0 +1,86 @@
+![HTML](https://img.shields.io/badge/-HTML-orange) ![CSS](https://img.shields.io/badge/-CSS-blue) ![JavaScript](https://img.shields.io/badge/-JavaScript-yellow)
+
+# To-Do List App
+
+A sleek and intuitive To-Do List app built with JavaScript. This app helps you organize your day, manage your tasks efficiently, and boost your productivity. Whether it's marking off simple daily chores or tackling big projects, this app will keep you on track!
+
+## 🌟 Features
+
+Task Creation: Effortlessly add your tasks and keep track of everything you need to get done.
+
+Task Completion: Mark tasks as completed with a simple checkbox and feel the satisfaction of checking them off.
+
+Edit and Modify: Update or change tasks on the fly with an easy-to-use editing feature.
+
+Task Deletion: Remove tasks that are no longer relevant with a quick delete button.
+
+Smart Filters: Filter tasks by completion status (All, Completed, Incomplete) for a more organized experience.
+
+Persistent Data: Tasks are saved locally using localStorage, ensuring your to-do list is always available when you return.
+
+Responsive Design: Perfectly fits on both mobile and desktop screens, making it easy to manage your tasks on the go.
+
+### Getting Started
+
+To get a local copy up and running follow these simple steps.
+
+Prerequisites
+VIsual Studio Code.
+A Web Browser (preferably Brave)
+
+### Setup
+
+Clone the GitHub Repository
+
+run this commands in your terminal:
+npm install
+npm run build
+npm start
+
+### Usage
+
+After running npm start the website will be opened automatically on your default browser.
+
```

Comment:
We don't need `npm start` to start the program. You can write that start your server by clicking `Go Live`